app/providers/repositories/user_token/index.js

```diff
@@ -7,18 +7,28 @@ class UserTokenRepository extends BaseRepository {
     return token
   }
 
-  async findTokenByUserId(userId, deviceId) {
-    const token = await this.findOne({ user_id: userId, device_id: deviceId })
+  async findAccessToken(jwtToken, deviceId, tokenType = "access") {
```

Comment:
Since we can pass `tokenType ` as an argument, it's better to name it as
`findAccessToken -> findToken`